dockers/actions-plan-preview/github.go

```diff
@@ -123,3 +127,107 @@ func getPullRequest(ctx context.Context, client *github.Client, owner, repo stri
 	pr, _, err := client.PullRequests.Get(ctx, owner, repo, prNum)
 	return pr, err
 }
+
+type issueComment struct {
+	ID     githubv4.ID
+	Author struct {
+		Login githubv4.String
+	}
+	Body        githubv4.String
+	IsMinimized githubv4.Boolean
+}
+
+type pullRequestCommentQuery struct {
+	Repository struct {
+		PullRequest struct {
+			Comments struct {
+				Nodes    []issueComment
+				PageInfo struct {
+					HasPreviousPage githubv4.Boolean
+					StartCursor     githubv4.String
+				}
+			} `graphql:"comments(last: 100, before: $commentsCursor)"`
+		} `graphql:"pullRequest(number: $prNumber)"`
+	} `graphql:"repository(owner: $repositoryOwner, name: $repositoryName)"`
+}
```

Comment:
[nit] imo, the shallower nest is better readability.

Reply:
I feel the same way.
I'll split comments into separate structs